guides/release/in-depth-topics/native-classes-in-depth.md

```diff
@@ -426,7 +426,7 @@ directly on the class itself.
 ```js
 class Alert {
   static helloWorld() {
-    alert('Hello, world!');
```

Comment:
I believe the `alert` refers to the built-in `Window.alert()` method. https://developer.mozilla.org/docs/Web/API/Window/alert

Based on lines 433 and 437, I think you are correct that `helloWorld` should be returning a string.

Reply:
That sounds right. I wasn't sure if that returns a string though, and there's no return statement so I thought maybe someone started off the example with an alert pop-up then decided to use console.log instead, and forgot to switch to returning the string.